artifactory/services/upload.go

```diff
@@ -70,11 +70,10 @@ func (us *UploadService) prepareUploadTasks(producer parallel.Runner, errorsQueu
 		defer producer.Done()
 		// Iterate over file-spec groups and produce upload tasks.
 		// When encountering an error, log and move to next group.
-		vcsDetails := clientutils.NewVcsDetals()
+		VcsCache := clientutils.NewVcsDetals()
```

Comment:
This variable starts with an upper case letter, which is not in line with the go convention.
This should also be fixed in functions which receive this variable.